data/json/snippets/effect_on_conditions.json

```
@@ -182,31 +182,53 @@
     "type": "snippet",
     "category": "PORTAL_STORM_VOICES_1",
     "text": [
-      "a gentle voice saying \"Why not come outside?\"",
-      "a gentle voice saying \"It's beautiful outside.\"",
-      "a gentle voice saying \"What a gorgeous sky.\"",
-      "a gentle voice saying \"Gosh the wind feels nice.\"",
-      "a gentle voice saying \"It would seem there are optimal human enjoyment conditions outside.\"",
-      "a gentle voice saying \"I'm so glad I came outside.\"",
-      "a gentle voice saying \"100% of those surveyed are very glad they came outside.\"",
-      "a gentle voice saying \"Maybe it isn't so bad outside after all, why don't you check?\"",
+      "a gentle voice saying \"Come outside.  I'm waiting for you.\"",
+      "a gentle voice saying \"Why are you not here yet?  It's beautiful outside.\"",
+      "a gentle voice saying \"What a sky.  Unlimited, deep, blue.  The ocean of fresh air is at hand while you gasp mold and dust behind four walls.\"",
+      "a gentle voice saying \"There is a place for you here.\"",
+      "a gentle voice saying \"The place you called home is not there anymore.  Come with me and find your new home.\"",
+      "a gentle voice saying \"What good comes from hiding from the truth in a mausoleum of brick walls?\"",
+      "a gentle voice saying \"There is no amount of concrete or stone you could put between you and the veil that is thinner then a shadow.\"",
+      "a gentle voice saying \"You want to sleep now?  Just moments after you have awaken us from our eternal slumber?\"",
```

```suggestion
      "a gentle voice saying \"You want to sleep now?  Just moments after you have awaken us from our eternal slumber?\"",
```
I think it would be awakened or awoke.  But I'm not 100% positive.

https://www.merriam-webster.com/words-at-play/usage-awaken-awoken-awakened

---

I'll double check when I'll have a chance to do so.

---

It's either, there are two words that are annoyingly similar.